Golden_Path.postman_collection.json

```diff
@@ -13101,107 +12939,112 @@
 						},
 						"url": {
-							"raw": "{{HOST_SWITCH_TRANSFERS}}{{BASE_PATH_SWITCH_TRANSFERS}}/transfers",
+							"raw": "{{HOST_QUOTING_SERVICE}}/quotes",
 							"host": [
-								"{{HOST_SWITCH_TRANSFERS}}{{BASE_PATH_SWITCH_TRANSFERS}}"
+								"{{HOST_QUOTING_SERVICE}}"
 							],
 							"path": [
-								"transfers"
+								"quotes"
 							]
 						}
 					},
 					"response": []
 				},
 				{
-					"name": "Check Send Quote 4 Decimal #949",
+					"name": "Check Send Transfer 4 Decimal #949",
 					"event": [
 						{
 							"listen": "test",
 							"script": {
-								"id": "4376c794-71a8-474c-9b25-a0714ca62df8",
+								"id": "900142b0-e4d7-43a4-a751-38202b600661",
 								"exec": [
 									"pm.test(\"Status code is 202\", function () {",
 									"    pm.response.to.have.status(202);",
 									"});",
 									"",
-									"//Check Data on payer side",
+									"//Check the request that Switch forwards to payeefsp",
 									"setTimeout(function () {",
-									"  pm.sendRequest(pm.environment.get(\"HOST_SIMULATOR\")+\"/payerfsp/callbacks/\"+pm.variables.get(\"quoteId\"), function (err, response) {",
-									"      if(response.responseSize !== 0) {",
-									"            ",
-									"            var jsonData = response.json().data;",
-									"            ",
-									"            pm.test(\"Send Quote Amount in 4 decimal returned\", function () {",
-									"            pm.expect(jsonData.transferAmount.amount).equal(\"60.1234\");",
-									"            });",
-									"            ",
-									"      } else {",
-									"          pm.test(\"Quote FAILED\", function () {",
-									"            throw new Error('Did not receive response');",
-									"           });",
+									"  pm.sendRequest(pm.environment.get(\"HOST_SIMULATOR\")+\"/payeefsp/requests/\"+pm.environment.get(\"transfer_ID\"), function (err, response) {",
+									"          if(response.responseSize !== 0) {",
+									"                ",
+									"                ",
+									"                //Check the data",
+									"                var jsonData = response.json().data;",
 									"",
-									"      }",
-									"       ",
+									"                pm.test(\"payeefsp data should have the same amount as request and 4 decimal checked\", function () {",
+									"                   pm.expect(jsonData.amount.amount).to.eql('1.9999');",
```

Comment:
Pls use the "amount" variable from the payload of the request in the assertions rather using a constant number.

Reply:
This will be dealt in #987(https://github.com/mojaloop/project/issues/987) and planned to have separate collection for bug fixes.

Reply:
Created a story to resolve this --> https://github.com/mojaloop/project/issues/1007